dashboard/app/views/levels/_free_response.html.haml

```diff
@@ -48,8 +48,7 @@
       ReactDOM.render(React.createElement(dashboard.Attachments, {showUnderageWarning: !appOptions.is13Plus, readonly: #{!!@view_options.readonly_workspace}}), document.querySelector('#free-response-upload'));
 
   - height = level.height || '80'
-  -# FND-984 placeholder needs to be localized level property.
-  - placeholder = level.placeholder || I18n.t('free_response.placeholder')
```

Comment:
I think we still want the `|| I18n.t('free_response.placeholder')` so the default text is still there. Most levels don't define the 'placeholder' property.

Reply:
oh, good catch!